### DISCUSSION TITLE
Font whitelist and clipboard.convert()

### DISCUSSION THREAD

**Discussion by TheEponymousBob** [slab/quill#4301] [Q&A]
Should adding a font to one's whitelist be reflected in `clipboard.convert()` for HTML when dealing with style attributes? I'm trying to render HTML from a legacy app that saves fonts as, for example, `<span style="font-family: roboto;">Hello</span>`.

I've updated the font whitelist and verified that it is going in to the registry. But when I trace through the `convert()` code, I reach `clipboard.matchAttributor()`, which checks a whitelist in `STYLE_ATTRIBUTORS`, and this does not seem to have been updated when the new font is registered.

I've put this in a [CodePen](https://codepen.io/crosswordbob/pen/rNEawdN?editors=1111). The console will show that only the monospace font is being recognised. Uncommenting the last two lines should show that the roboto font has been registered.

**Comment by MuhammadOwais-SE** [comment]
Faceing the issues. I can not able to add any font in using next.js
